src/ethereum/index.ts

```diff
@@ -176,9 +180,17 @@ export default class EthereumWallet implements IWallet {
 
     logger.info('Eth balance', { ethBalance });
 
+    let l1fees = new BigNumber(0);
+    if (
+      this.coin.coinListId === ETHCOINS[EthCoinMap.optimism].coinListId &&
+      l1Cost
+    ) {
+      l1fees = new BigNumber(l1Cost).multipliedBy(this.optimismL1MaxVariation);
```

Comment:
This hike should be in the server or just after we get the L1 cost. So that this logic does not have to be repeated.

It'll also eliminate the `l1fees` variable which is confusing with `l1cost`

Reply:
This calculation is done on cysync because in the future we might want to tune this variation, to scale down the amount left in the account for send max.
The repeated code exists due to the reason that the fee calculation function already exists but is not being used in creating transactions. This could be solved by a refactor of the function but I don't think that's in scope of this PR

Reply:
Moved the multiplication to useSendTransaction but the l1fees variable is required to return the l1Cost as a BigNumber